frontend/javascripts/admin/dataset/dataset_upload_view.tsx

```diff
@@ -91,9 +99,12 @@ type State = {
   isRetrying: boolean;
   uploadProgress: number;
   selectedTeams: APITeam | Array<APITeam>;
-  uploadId: string;
+  possibleTeams: Array<APITeam>;
+  uploadId: string | null;
```

Comment:
it is still initialized to an empty string. can be changed to null now probably.

Reply:
:thinking:, down in the code I do explicitly set the `uploadId` to null to make it explicit that a new one should be generated after a successful or failed upload or in case no previous existing upload was selected from the list. Using `""` as a placeholder seems a little to implicit to me. I am aware that js evaluates `""` as false this is too implicit for me.

While writing this I came up with the following: I made the default value of uploadId undefined as a new uploadId should be generated when none is defined. This includes the initial value and the value set after an upload.  

Does that make sense? In short: I prefer to make the state of "don't use an already ongoing upload" more explicit via an undefined than an empty string :)

But feel free to argue

Reply:
I meant [line 189](https://github.com/scalableminds/webknossos/pull/7981/files/7730af5293635cccbf5b2f61411bbd64e2d3bbcf#diff-1ad4a75b67f7779e859eb39c32a1e3bd5d4c220c5146b4b1089fbe36014d8d42R189) where `uploadId` is set to `""`. that should probably be changed to `null`.

> Using "" as a placeholder seems a little to implicit to me.

yes, i fully agree.

Reply:
Oh sorry big misread from my side. Concentration kinda slipped I guess. I read it like: "Please change all `uploadId: null` assignments to `uploadId: ""`  :see_no_evil:

Reply:
it's good that you didn't agree to that :sweat_smile: